src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles (see an example
+ * in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const styles = sourceString.split(';').map(str => str.trim().split(':'));
```

Comment:
[Checklist](https://github.com/mate-academy/js_style-to-object/blob/master/checklist.md) 4th item, use proper indentation for your method calls
```suggestion
  const styles = sourceString
    .split(';')
    .map(str => str.trim()
    .split(':'));
```